samples/src/components/Samples/AtMountTimeout/index.js

```diff
@@ -0,0 +1,19 @@
+import React, { PropTypes } from 'react'
+import animate from 'hoc-react-animate'
+
+import styles from '../animate.scss'
+
+const AtMount = ({ ex, className }) => (
+  <div className={`${styles.component} ${className}`}>
+    I'm animate at mount.<br />
```

Comment:
"Animate it" c'est bien pour un bouton mais je n'ai pas l'impression que ce soit le cas ici.